src/index.js

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './components/App';
 import * as serviceWorker from './serviceWorker';
+import booksReducer from './reducers/index';
+
+const state = {
+  books: [
+    { id: Math.floor(1000 * Math.random(5, 5)), title: 'Lolita', category: 'Romance' },
+    { id: Math.floor(1000 * Math.random(5, 5)), title: 'Lolita', category: 'Romance' },
+    { id: Math.floor(1000 * Math.random(5, 5)), title: 'Lolita', category: 'Romance' },
+    { id: Math.floor(1000 * Math.random(5, 5)), title: 'Lolita', category: 'Romance' },
+    { id: Math.floor(1000 * Math.random(5, 5)), title: 'Lolita', category: 'Romance' },
```

Comment:
**OPTIONAL**

- [ ] You can use uuidv4 to automatically generate ids for you. Check the package [here](https://www.npmjs.com/package/uuidv4).
```javascript
import uuid from 'uuidv4'

 { id: uuid(), title: 'Lolita', category: 'Romance' }, //generates an unique uuid.
```

Reply:
Thanks for the suggestion. 👍